cmds/core/init/init.go

```diff
@@ -13,18 +13,14 @@ package main
 import (
 	"flag"
 	"fmt"
-	"io/ioutil"
 	"log"
 	"os"
 	"os/exec"
-	"path"
-	"path/filepath"
-	"runtime"
-	"strings"
 	"syscall"
 
+	"github.com/u-root/u-root/pkg/libinit"
```

Comment:
is pkg/libinit kind of redundant, pkg being a kind of standin for "lib" anyway? is pkg/init ok?

Reply:
no, because `init` is a reserved word for init functions